Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -373,8 +373,8 @@
     <string name="risk_details_explanation_dialog_title">"Information zur Funktionsweise der Risiko-Ermittlung"</string>
     <!-- YTXT: one time risk explanation dialog - pointing to the faq page for more information-->
     <string name="risk_details_explanation_dialog_faq_body">"Weitere Informationen finden Sie in den FAQ."</string>
-    <!-- XLNK: risk explanations and informations - pointing to the faq page for more information and contains hyperlink-->
-    <string name="risk_details_explanation_faq_body_with_link"><a href="https://www.coronawarn.app/de/faq/#encounter_but_green">Weitere Informationen finden Sie in den FAQ.</a></string>
+    <!-- XLNK: FAQ URL pointing to the faq page in german. Need to use the URL for english for all other languages-->
+    <string name="risk_details_explanation_faq_body_with_link">"https://www.coronawarn.app/de/faq/#encounter_but_green"</string>
```

Comment:
Not sure we can reuse the same key. Probably better to introduce a new one. How was this handled in the past @harambasicluka ?

Reply:
I think it's fine if @ritsam checked that it's not used somewhere else.